german/the-black-hack.md

```diff
@@ -231,15 +232,15 @@ Einige Monster und NSCs haben festgelegte Persönlichkeiten und Ziele, die dem S
 | 7    | Ruft Verstärkung.                    |
 | 8    | Will die SCs fangen/töten/fressen.   |
 
-## Untote Bannen
+## Untote bannen
 
 Kleriker können versuchen, mit einer Handlung alle Untoten in **Schrittweite** zu bannen. Dazu müssen sie für jede Gruppe von Wesen, die sie bannen wollen, eine Probe auf **WEI** bestehen, wobei der **TW** der Wesen zum Wurf addiert wird. Der SL legt fest, welche Wesen zu einer Gruppe gehören.
 
-Untote Monster, die die Kleriker **Bannen** konnten, müssen all ihre Bewegungen dazu benutzen (und all ihre Handlungen in Bewegungen verwandeln), um sich für 2W4 **Momente** vom Kleriker zu entfernen, nachdem sie von **Bannen** betroffen wurden.
+Untote Monster, die die Kleriker **bannen** konnten, müssen alle ihre Bewegungen dazu benutzen (und alle Handlungen in Bewegungen umwandeln), um sich für 2W4 **Momente** vom Kleriker zu entfernen, nachdem sie von **Bannen** betroffen wurden.
 
 ## Vorratswürfel
 
-Alle Gegenstände im Abschnitt zu Ausrüstung, für die ein **Vorratswürfel** angegeben ist, sind Verbrauchsgegenstände mit begrenzter Anzahl. Wenn solch ein Gegenstand in der nächsten **Minute** (Runde) benutzt wurde, wird sein **Vorratswürfel** gewürfelt. Wenn das Ergebnis 1-2 ist, wird der Vorratswürfel nach dem folgenden Schema verringert:
+Alle Gegenstände im Abschnitt zu Ausrüstung, für die ein **Vorratswürfel** angegeben ist, sind Verbrauchsgegenstände mit begrenzter Anzahl. Wenn solch ein Gegenstand benutzt wurde, wird in der nächsten **Minute** (Runde) sein **Vorratswürfel** gewürfelt. Wenn das Ergebnis 1-2 ist, wird der Vorratswürfel nach dem folgenden Schema verringert:
```

Comment:
Habe es mal so verbessert, wie ich die Vorlage verstehe:

> When that item is used the next Minute (turn) its Usage die is rolled.

Ich benutze eine Fackel (die ich im Rucksack habe). Dann würfle ich in der folgenden Runde, ob ich sie noch habe.

Alternativ kann ich es auch lesen:

Wenn ich in der folgenden Runde eine Fackel verwenden will, würfle ich vorher, ob sie noch da ist.

Die zweite Regelung fände ich etwas komisch - dann würde ich eventuell tagelang eine Fackel mit mir herumschleppen, die ich aufgeschrieben habe und die Platz wegnimmt, um dann beim Anzünden plötzlich zu merken, dass sie gar nicht mehr da ist?